redhawk/common/node.py

```diff
@@ -1234,6 +1235,23 @@ def GetSExp(self):
     return li
 
 
+class Try(TryCatch):
```

Comment:
Kind of. Python 3 `ast` parses try statement as just `Try` (no `TryExcept` or `TryFinally`) - even if parsing Python 2 code. It is true that this also moves us closer to implementing #2, but this is needed even if parsing Python 2 code while running redhawk on Python 3.